sklearn/ensemble/iforest.py

```diff
@@ -181,10 +186,37 @@ def fit(self, X, y=None, sample_weight=None):
         super(IsolationForest, self)._fit(X, y, max_samples,
                                           max_depth=max_depth,
                                           sample_weight=sample_weight)
+
+        self.threshold_ = -sp.stats.scoreatpercentile(
+            -self.decision_function(X), 100. * (1. - self.contamination))
+
         return self
 
     def predict(self, X):
-        """Predict anomaly score of X with the IsolationForest algorithm.
+        """Outlyingness of observations in X according to the fitted model.
```

Comment:
Based on the output I think this should be `'Predict if a particular sample is an outlier or not'`?